frontend/src/libs/hooks/use-articles-feed-room/use-articles-feed-room.hook.ts

```diff
@@ -40,8 +47,24 @@ const useArticlesFeedRoom = (): void => {
     articlesSocket?.on(
       NEW_ARTICLE,
       (newArticle: ArticleSocketEventPayload[typeof NEW_ARTICLE]) => {
-        if (location.pathname === AppRoute.ARTICLES) {
-          void dispatch(articleActions.addArticle(newArticle));
+        if (user?.id === newArticle.socketUserId) {
```

Comment:
I am not sure why we need to send `socketUserId` in the event's payload, can't it be 
`if (user?.id === newArticle.article.id) return` ? Maybe I misunderstood something 😅

Reply:
In this case we only check the author of the article, but on the server we loop through all the sockets in the namespace and send to each user info about each socket on every loop cycle (for example if we have 5 sockets each user will get 5 payloads), so if only one user follows the author, every user will be notified of this publication due this one payload from that user.

Reply:
Well, maybe it could have been done better, but I was in a bit of a rush to meet the deadline)